app/src/renderer/js/main.js

```diff
@@ -55,8 +53,11 @@ document.addEventListener('DOMContentLoaded', () => {
   let monitoringIntervalId;
   let lastValidInputWidth = 512;
   let lastValidInputHeight = 512;
-  let aspectRatioBaseValues = [lastValidInputWidth, lastValidInputHeight];
   let hasUpdateNotification = false;
+  let height = 512;
+  let width = 512;
+  let selectedRatio = '1:1';
+  let lockedRatio = false;
```

Comment:
Do these really need to be global variables? And if yes, I think they at least should be scoped in an object so it's  clearer where they belong.

Reply:
They're used in multiple places so I think they have to be. Will assign them to an object for cleanliness 👌